package.json

```diff
@@ -2,10 +2,11 @@
   "name": "smile-identity-core",
   "version": "2.0.0",
   "description": "The official Smile Identity Node SDK",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "lint": "eslint $(git ls-files '*.js')",
-    "start": "node index.js",
+    "build": "tsc",
+    "lint": "eslint $(git ls-files '*.js' '*.ts')",
+    "start": "node dist/index.js",
```

Comment:
This may not exist. You should take another pass on setting up scripts.

Reply:
done, this is now generated when npm install is run